src/Form/AuthenticationForm.php

```diff
@@ -145,13 +159,16 @@ protected function getEditableConfigNames() {
   public function buildForm(array $form, FormStateInterface $form_state) {
     $form = parent::buildForm($form, $form_state);
 
-    $form['#prefix'] = '<div id="apigee-edge-auth-form">';
-    $form['#suffix'] = '</div>';
     $form['#attached']['library'][] = 'apigee_edge/apigee_edge.admin';
 
     // Save the key for later use.
     $this->activeKey = $active_key = $this->getActiveKey();
 
+    // Placeholder for messages.
+    $form['messages'] = [
+      '#markup' => '<div id="apigee-edge-auth-form-messages"></div>',
```

Comment:
I'd rather use the "html_tag" render element to prepare this placeholder for messages.

Reply:
But it is much easier to alter in a 3rd party module, because it becomes html_tag is a render element where the value and how the value is rendered is separated from each other. (Probably this form is never going to be altered, but we can not be sure about that. I always prefer to use html_tag instead of markup every time when markup's value would contain html tags.